src/core/geometry/geom3/clone.js

```diff
@@ -5,9 +5,11 @@ const poly3 = require('../poly3')
  * @param  {Geom3} sourceGeometry the geometry to clone
  * @returns {Geom3} the new clone
  */
-const clone = sourceGeometry => {
+const clone = (sourceGeometry) => {
   return {
-    polygons: sourceGeometry.polygons.map(p => poly3.clone(p)),
+    polygons: sourceGeometry.polygons.map((p) => {
```

Comment:
why the change here ? the previous was a perfectly valid syntax ?

Reply:
eslint warnings

i won't change back now because we haven't setup lint. there will be several rounds of cleanup ahead.